great-idea/css/index.css

```diff
@@ -63,4 +63,353 @@ h1, h2, h3, h4, h5 {
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+.container {
+  width: 880px;
+  /* TB LR */
+  margin: 0 auto;
+}
+
+/* header */
+
+header nav {
+	display: flex;
+	width: 691px;
+	vertical-align: top;
+	margin-top: 35px;
+	width: 100%;
+  }
+  
+  header nav a {
+	display: flex;
+	margin-right: 65px;
+	font-size: 14px;
+	color: gray;
+	text-decoration: none;
+  }
+
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header nav a:first-child {
+  margin-left: 0;
+}
+
+header nav a:last-child {
+  margin-right: 0;
+}
+
+header .logo {
+	display: flex;
+	float: right;
+	position: relative;
+	  right: 18px;
+	  bottom: 30px;
+	margin-bottom: 0;
+	padding-left: 200px;
+  
+  }
+
+/* cta */
+
+.cta .cta-text {
+	flex-direction: row;
+	font-size: 77px;
+	margin-top: 120px;
+	text-align: center;
+	padding: 0px 75px;
+	padding-right: 560px;
+}
+
+.cta .cta-text button {
+	background: white;
+	vertical-align: top;
+	margin-top: 6px;
+	padding: 8px 44px;
+	font-size: 17px;
+	border: 1px solid black;
+	cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+
+
+}
+
+.cta img {
+	display: flex;
+	float: right;
+	position: relative;
+	top: -350px;
+
+}
+
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 340px;
+  padding-top: 0px;
+  position: relative;
+  top: -240px;
+  
+
+}
+
+.main-content .top-content {
+	padding: 0 10px;
+	float: right;
+
+}
+
+
+.main-content .top-content .text-content {
```

Comment:
I like how specific you were in this section, all of your tags should be this specific. It helps to do it this way because it lets you reuse the tag if you have two pages and want to use a common class name like `.text-content`.
in this example you used .main-content .top-content .text-content, later on you could do .second-content .mid-content .text-content and the text-content would be free from the code you wrote on the first one because of how specific you were. Keep it up.